Automattic/woocommerce-payments
# Avoid returning Stripe error message when capturing more than order amount

Fixes #7029 

#### Changes proposed in this Pull Request









* Redact the prompt in order notes to contact Stripe support when a merchant attempts to capture more than the authorized amount. Contacting support is unnecessary in this case.
* Introduce `Amount_Too_Large_Exception` API exception

**Before**

<img width="277" alt="261541727-6f87fd67-9afa-462b-a7fa-1a693c0b2339" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/5b5f5519-8916-4f8a-baf3-8163130dfce2">

**After**

<img width="290" alt="Screenshot 2023-11-02 at 10 11 47" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/422d2e4f-2b80-4023-b95a-03698291c4e3">

#### Testing instructions




- As a merchant, enable the authorize and capture later setting under Payments > Settings.
- As a shopper, place an order
- As a merchant, edit the order to add a fee or use any other method to increase the order total beyond its original amount.
- Using order actions box, try to capture the order.
- Ensure there's no prompt advising the merchant to contact Stripe support (refer to the "After" screenshot above).


-------------------


**Post merge**
